src/providers/AppProvider.tsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
 
-type AppProviderProps = {
-  children: React.ReactNode;
-};
+import { JustChildrenProp } from '../types';
```

Comment:
not asking for this PR but you might want to configure absolute imports soon so there is little code you have to convert to the new import style. It makes coding much easier.

Reply:
That’s a great suggestion, I’ve already added it here #3 :)